applications/GeoMechanicsApplication/custom_utilities/process_utilities.cpp

```diff
@@ -16,6 +16,32 @@
 #include "containers/model.h"
 #include "includes/kratos_parameters.h"
 
+namespace
+{
+std::set<std::string, std::less<>> ExtractModelPartNames(const auto&      rProcessList,
+                                                         std::string_view RootName,
+                                                         std::string_view Prefix)
+{
+    std::set<std::string, std::less<>> result;
```

Comment:
Not blocking, but could this be simplified? 
```suggestion
std::set<std::string> ExtractModelPartNames(const auto&      rProcessList,
                                                         std::string_view RootName,
                                                         std::string_view Prefix)
{
    std::set<std::string> result;
```

Reply:
they are added to remove code smells